### DISCUSSION TITLE
Bubbles for Design Pages/Ambassodor needed calls to action

### DISCUSSION THREAD

**Discussion by bobgeorgethe3rd** [freesewing/freesewing#1781] [Forum]
Cross-posted from discord.
This is just an idea that I had when admiring the new dev site for the org site.
One of the things I love is the little bubbles as it shows of the community. I was thinking what if we use those to display the designers, coders and ambassadors on the design pages with a little tag with ign and contribution title (coder,designer,ambassador). Then what we can do is have an Ambassador needed bubbles for patterns that need one. Kinda like a "Love this pattern? It needs an ambassador to help take care of it" etc that links to the Ambassadors pages etc.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
